streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamThreadTest.java

```diff
@@ -2997,27 +2863,19 @@ void runOnceWithoutProcessingThreads() {
 
         thread.setState(StreamThread.State.STARTING);
         thread.runLoop();
-
-        verify(taskManager);
-        verify(consumer);
     }
 
     @Test
     @SuppressWarnings("unchecked")
     public void shouldNotEnforceRebalanceWhenTaskCorruptedExceptionIsThrownForAnInactiveTask() {
         final StreamsConfig config = new StreamsConfig(configProps(true));
-
-        final TaskManager taskManager = EasyMock.createNiceMock(TaskManager.class);
-        expect(taskManager.producerClientIds()).andStubReturn(Collections.emptySet());
+        final TaskManager taskManager = mock(TaskManager.class);
+        when(taskManager.producerClientIds()).thenReturn(Collections.emptySet());
```

Comment:
I think you can remove this since it is the default return value and you removed those stubs in other places.